DuckDuckGo/FireAnimation.swift

```diff
@@ -19,56 +19,50 @@
 
 import UIKit
 
-struct FireAnimation {
-    
+extension FireAnimation: NibLoading {}
+
+class FireAnimation: UIView {
+
+    @IBOutlet var image:UIImageView!
+    @IBOutlet var offset:NSLayoutConstraint!
+
     struct Constants {
-        static let animationDuration = 1.4
-        static let endAnimationDuration = 0.1
-        static let initialFirePeekPortion: CGFloat = 0.15
+        static let animationDuration = 1.9
+        static let endDelayDuration = animationDuration + 0.2
+        static let endAnimationDuration = 0.9
```

Comment:
Three seconds feels like a long a delay to me. When this task was originally prototyped we thought 2 seconds was a bit long. Can you take it back to the previous values? e.g
```
static let animationDuration = 1.4 or 1.5
static let endDelayDuration = animationDuration 
static let endAnimationDuration = 0.1 or 0.2
```

Fyi the animator designed the flames to work best in a 1-2s animation. It feels about right on a medium device but a bit slow on a tiny device and a bit fast on a tablet.

If you're happy to make that change, consider this approved! Merge when you're ready.

Reply:
Did you try changing these? 

I don't mind making them smaller, but I tried smaller values and it just didn't look as slick.  On a device (IMHO) these look really good.  Let me know if you tried them (and on a device) and if so, I'm happy to change them.

RE: `static let endDelayDuration = animationDuration + 0.2`

The 0.2 here is required to prevent a "flash" when the underlying screen changes back to the home screen.

Reply:
Also worth noting that the previous implementation with the nested animation to remove the view effectively did nothing as removing a view from the superview can't be animated.  So any additional animation to tidy that up was going to add more time to the overall experience.